term_proposal/cthbst/proposal.md

```diff
@@ -0,0 +1,20 @@
+# Particle collision simulation
+
+## Introduction
+
+Simulate $N$ colliding particles help scientists to verifying some basic law in microscopic scale.
+
+In this project, our goal is to simulate at least 1000 particles in realtime (>30fps).
```

Comment:
In molecular dynamics the number of particles is much more important than the frame rate.  I don't think you are doing molecular dynamics per se, but still scalability matters more than responsiveness.

Could you address also in the project that how can you scale up?